### PR TITLE
Created pull request template.

### PR DESCRIPTION
**Pull request by HassanJaveed84**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Analytics Pipeline Pull Request
+---
+
+Make sure that the following steps are done before merging:
+
+  - [ ] If you have a migration please contact data engineering team before merging.
```
Comment (brianhw) [inline .github/pull_request_template.md:6]: Do we want to list a way of contacting the data engineering team in this PR?   Is there a tag or an email address, or should we just leave it to people to figure this out?

Also nit:  "contact the data engineering team" sounds better to my ear, both here and in the third checkbox.

Comment (HassanJaveed84) [reply .github/pull_request_template.md:6]: Do you have any suggestions on what needs to be mentioned here ? Do we have a tag for DE team or should we mention a specific person tag ?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  - [ ] Before merging run full acceptance tests suite and provide URL for the acceptance tests run.
+  - [ ] A member of data engineering team has approved the pull request.
```
Comment (HassanJaveed84) [inline .github/pull_request_template.md:8]: Should this be 'Contact the data engineering team before merging?'

File: .github/pull_request_template.md (hunk 1 above)
Comment (brianhw) [reply .github/pull_request_template.md:6]: I don't have a particular suggestion, which is why I asked a question. : )   But I definitely think we should not mention a specific person.   The migration needs to be raised more broadly.    If we don't have a group tag or email, then we can leave the onus on the PR author to figure out how to reach us.   I'm fine with the existing wording as being a good step forward.

File: .github/pull_request_template.md (hunk 2 above)
Comment (brianhw) [reply .github/pull_request_template.md:8]: I liked your original sentiment that DE needs to approve, not just be notified.   That way DE can at least acknowledge that they were notified.

**Review by brianhw — APPROVED** [review]
I'm fine with it as-is, if there's no obvious contact info we want to add.

**Review by azafty468 — APPROVED** [review]
👍 LGTM!

**Pull request merged**